doc/README.md

```diff
@@ -322,6 +323,59 @@ config:
     disco_srv: http://disco.example.com
 ```
 
+##### Mirror the SAML ForceAuthn option
+
+By default when the SAML frontend receives a SAML authentication request
+with `ForceAuthn` set to `True`, this information is not mirrored in the SAML
+authentication request that is generated by the SAML backend towards the
+upstream identity provider. If the configuration option
+`mirror_force_authn` is set to `True`, then the default behaviour changes
+and the SAML backend will set `ForceAuthn` to true when it proxies a SAML
+authentication request with `ForceAuthn` set to `True`.
+
+The default behaviour is `False`.
+
+```yaml
+config:
+  mirror_force_authn: True
+  [...]
+```
+
+##### Memorize the IdP selected through the discovery service
```

Comment:
This has now changed, and happens regardless of whether the selection came from the discovery service.
There is actually, no other selection mechanism other than the discovery service. In all other cases there is either only one IdP (no choice), or the IdP has been chose as part of the mirrored-frontend (see `Context.KEY_TARGET_ENTITYID` - again, no choice).